# TODO(Issue #8): Will need to tune min and max height to try to get to as close
#                 as possible to pitch of 0 degrees.
min_height: -0.3
max_height: 0.3

# (Almost) All the rest of these are default
# Use the default values for angle params (but don't know how to specify pi in
# yaml file so leaving commented out for now
angle_min: -π/2
angle_max: π/2
angle_increment: π/360
scan_time: 1.0/30.0
range_min: 0.45
range_max: 4.0
concurrency_level: 0 # Auto-detect and use number of cores available
use_inf: true

